tests/aws/services/lambda_/functions/common/README.md

```diff
@@ -0,0 +1,29 @@
+# Lambda Multiruntime Builds
+
+This directory contains the source code and build instructions for Lambda multiruntime tests.
+Example tests are available under `tests.aws.lambda_.functions.common`
+
+Each scenario (e.g., "echo") has the following folder structure: `./<scenario>/runtime/`
+A runtime can be an aggregated runtime defined in `runtimes.py` (e.g., "python") or
+a specific runtime (e.g., "python3.12") if customizations are required.
+
+Each runtime directory defines a `Makefile` that
+* MUST define a `build` target that:
+  * a) creates a `build` directory containing all Lambda sources ready for packaging
+  * b) creates a `handler.zip` file with a Lambda deployment package
+* SHOULD define an `ARCHITECTURE` parameter to overwrite the target architecture (i.e., `x86_64` or `arm64`)
+  if architecture-specific builds are required (e.g., Dotnet, Golang, Rust).
+  * By default, the Makefile should build a deployment package with the same architecture as the host.
```

Comment:
This default behavior is debatable:
* It currently aims for compatibility. For example, the build on a Linux ARM machine should not fail.
* It differs from the default Lambda behavior using `x86_64` by default.

Hence, on multi-architecture platforms (e.g., Apple Silicon), x86_64 builds require a parameter `make ARCHITECTURE=x86_64`.